framework/cordova.gradle

```diff
@@ -168,11 +152,17 @@ def doGetConfigPreference(name, defaultValue) {
 
 // Properties exported here are visible to all plugins.
 ext {
+    def jsonFile = new File("$rootDir/config.json")
```

Comment:
This needs to be confirmed.

Previously it was:
```
def jsonFile = new File("$rootDir/${getProperties().get('cordova.defaultsFileLocation')}/defaults.json")
```

I have tested:

* `npm t`
* Creating a project from CLI
* Building project from CLI

I have not confirmed with:
* platform-centered workflow

I believe this workflow was going to be removed in this release but raphinesse will need to confirm this as he has the PR in draft.

Reply:
I never tested platform-centered workflow either with my original PR draft.